eventrecorder/eventrecorderpb/eventrecorder.proto

```
@@ -0,0 +1,392 @@
+syntax = "proto3";
+
+package eventrecorder;
```

Buf (1.66.1)

[error] Files with package "eventrecorder" must be within a directory "eventrecorder" relative to root but were in directory ".".

(PACKAGE_DIRECTORY_MATCH)